### PR TITLE
New Package: walld-1.4.1

### PR DESCRIPTION
**Pull request by Dotz0cat**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (amd64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - i686

### CODE REVIEW

File: srcpkgs/walld/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+archs="i686 x86_64"
+build_style=meson
+hostmakedepends="pkg-config"
+makedepends="libconfig-devel libevent-devel libmagick-devel"
+depends="feh"
+short_desc="Wallpapper daemon"
+maintainer="Dotz0cat <Dotz0cat@gmail.com>"
+license="GPL-3.0-only"
+homepage="https://github.com/Dotz0cat/walld"
+distfiles="https://github.com/Dotz0cat/walld/archive/refs/tags/${version}.tar.gz>${pkgname}-${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/walld/template:14]:
```suggestion
distfiles="https://github.com/Dotz0cat/walld/archive/refs/tags/${version}.tar.gz"
```
should work fine without renaming

File: srcpkgs/walld/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'walld'
+pkgname=walld
+version=1.4.1
+revision=1
+archs="i686 x86_64"
+build_style=meson
+hostmakedepends="pkg-config"
+makedepends="libconfig-devel libevent-devel libmagick-devel"
+depends="feh"
+short_desc="Wallpapper daemon"
```
Comment (classabbyamp) [inline srcpkgs/walld/template:10]: typo? (papper)

File: srcpkgs/walld/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'walld'
+pkgname=walld
+version=1.4.1
+revision=1
+archs="i686 x86_64"
```
Comment (classabbyamp) [inline srcpkgs/walld/template:5]: this will probably build on all arches, let's drop the restriction for now

Comment (Dotz0cat) [reply srcpkgs/walld/template:5]: It should on all glibc arches. So I will set it as ~*-musl

Comment (paper42) [reply srcpkgs/walld/template]: Why does it not build on musl? That should be fixed with a patch and sent upstream.

Comment (Dotz0cat) [reply srcpkgs/walld/template]: It reassigned stdin, stdout, and stderr. There is a patch that just closes them. It will be included in upstream 1.4.2.

**Comment by paper42** [comment]
I see this is your project, noone else requested this package and it only has 13 stars. Every package increases the maintenance effort needed by void maintainers and I think this one is just not worth it if noone is going to use it. Sorry, but this package is getting a NACK from me.

**Pull request closed without merging**